packages/common/src/pipes/date_pipe.ts

```diff
@@ -18,6 +19,13 @@ import {invalidPipeArgumentError} from './invalid_pipe_argument_error';
  */
 export const DATE_PIPE_DEFAULT_TIMEZONE = new InjectionToken<string>('DATE_PIPE_DEFAULT_TIMEZONE');
 
+/**
+ * Optionally-provided default format to use for all instances of `DatePipe` (such as `short`).
+ * If the value isn't provided, the `DatePipe` will default to 'mediumDate'.
+ */
+export const DATE_PIPE_DEFAULT_OPTIONS =
```

Comment:
Let's add the public API annotation to this new symbol:
```suggestion
/**
 * Optionally-provided default format to use for all instances of `DatePipe` (such as `short`).
 * If the value isn't provided, the `DatePipe` will default to 'mediumDate'.
 *
 * @publicApi
 */
export const DATE_PIPE_DEFAULT_OPTIONS =
```